code/modules/food_and_drink/snacks.dm

```diff
@@ -207,7 +207,61 @@
 				src.topping_holder -= topping_color
 				I.Blend(topping_color, ICON_ADD)
 				src.overlays += I
+/obj/item/reagent_containers/food/snacks/pizza/random
+	desc = "A pizza that somehow generates its own toppings and cooks itself! Oh the marvels of technology"
```

Comment:
tone

Reply:
No one should ever see that so I thought it was in tone for an internal description, should I do a "you shouldn't see this!" message?